### ISSUE TITLE
create phalcon.ini  Directory nonexistent

### ISSUE THREAD

**Issue by echolevan** [laradock/laradock#1844]
### Info:
- Docker version (`$ docker --version`): Docker version 18.06.1-ce, build e68fc7a
- Laradock commit (`$ git rev-parse HEAD`): 9cfe5c61565a77b4da40817f3e1d45e18ef5920d
- System info (Mac, PC, Linux): MAC

### Issue:

_____

### Expected behavior:

_____

### Reproduce:

_____

### Relevant Code:

```
/bin/sh: 1: cannot create /etc/php/7.2/mods-available/phalcon.ini: Directory nonexistent
ERROR: Service 'php-fpm' failed to build: The command '/bin/sh -c if [ $INSTALL_PHALCON = true ]; then apt-get update && apt-get install -y unzip libpcre3-dev gcc make re2c && curl -L -o /tmp/cphalcon.zip https://github.com/phalcon/cphalcon/archive/v${LARADOCK_PHALCON_VERSION}.zip && unzip -d /tmp/ /tmp/cphalcon.zip && cd /tmp/cphalcon-${LARADOCK_PHALCON_VERSION}/build && ./install && echo "extension=phalcon.so" >> /etc/php/${LARADOCK_PHP_VERSION}/mods-available/phalcon.ini && ln -s /etc/php/${LARADOCK_PHP_VERSION}/mods-available/phalcon.ini /etc/php/${LARADOCK_PHP_VERSION}/cli/conf.d/30-phalcon.ini && rm -rf /tmp/cphalcon* ;fi' returned a non-zero code: 2
```

**Comment by vladyslavstartsev** [comment]
@kideny Can you please help? Or that's the thing you fixed in #1811 ?

**Comment by kideny** [comment]
yes，i fixed it in #1811
